cache/compressed_secondary_cache.cc

```diff
@@ -149,7 +149,6 @@ CompressedSecondaryCache::SplitValueIntoChunks(
 
   CacheValueChunk dummy_head = CacheValueChunk();
   CacheValueChunk* current_chunk = &dummy_head;
```

Comment:
The changes in this file are in https://github.com/facebook/rocksdb/pull/10507 . This PR should be landed after https://github.com/facebook/rocksdb/pull/10507